Plugins/Public/base_plugin/Main.h

```diff
@@ -576,8 +576,6 @@ extern float set_shield_damage_multiplier;
 /// Holiday mode
 extern bool set_holiday_mode;
 
-wstring HtmlEncode(wstring text);
```

Comment:
This one seems suspect to me. It likely isn't appropriate for all use cases.

Reply:
I looked at the function, I don't see anything _glaringly_ off about it.